src/net.h

```diff
@@ -982,7 +982,13 @@ class CConnman
      */
     bool AlreadyConnectedToAddress(const CAddress& addr);
 
-    bool AttemptToEvictConnection();
+    /**
+     * Attempt to find a connected node to disconnect.
+     * Used to make room for new inbound connections, returns true if successful.
+     * @param[in] force     Try to evict a random node if all connections are otherwise protected.
```

Comment:
suggested rephrasing (although I'm not sure about the "NoBan" qualifier, maybe we have a better established term?)
```suggestion
     * @param[in] force     If all connections are otherwise protected, still evict a random inbound NoBan node if it exists
```